R/calcProjectPipelines.R

```diff
@@ -0,0 +1,66 @@
+#' calc Project Pipelines
```

Comment:
Please make sure to note where this will be called. When I clean up old code, I usually run sth like
`madrat::getDependencies("calcProjectPipelines", direction = "dout")`  and if I find no results, I might just delete the code.

Reply:
Edit: just read you comment concerning this. I am fine with the redundancy in this case. It gives you flexibility, should requirements by the different teams diverge in the future.

Reply:
Will do, so far it is not called anywhere, but I assume sooner or later it will be used either in ``fullValidation`` or some similar new function.